### DISCUSSION TITLE
大佬们，onlyoffice不会搞https的预览，求助。

### DISCUSSION THREAD

**Discussion by monobo1989** [AlistGo/alist#7300] [Q&A]
[https://blog.csdn.net/weixin_39504040/article/details/139364433?ops_request_misc=%257B%2522request%255Fid%2522%253A%25228CB83F8D-F8F2-4FE9-BBAC-19C77D7DE10C%2522%252C%2522scm%2522%253A%252220140713.130102334..%2522%257D&request_id=8CB83F8D-F8F2-4FE9-BBAC-19C77D7DE10C&biz_id=0&utm_medium=distribute.pc_search_result.none-task-blog-2~all~baidu_landing_v2~default-5-139364433-null-null.142^v100^pc_search_result_base5&utm_term=alist%20onlyoffice](url) 
我按照以上教程弄好了http的预览，lframe预览如下图

![微信图片_20241006140239](https://github.com/user-attachments/assets/dd389bc0-f7a0-4eb9-998a-f1d00de6d2c6)

我把alist搭建在甲骨文服务器上了，用域名访问，但是现在用caddy反代不成功。求大佬指导。
caddyfile内容：
![微信图片_20241006135935](https://github.com/user-attachments/assets/de15345a-7efc-4376-afad-2cd26025e539)
域名通过CF解析到caddy所在的服务器：
![微信图片_20241006140040](https://github.com/user-attachments/assets/307de1e8-d5d5-4a12-85aa-e9af2f0ba262)
lframe预览如下图
![微信图片_20241006140411](https://github.com/user-attachments/assets/a9cc9ae0-769b-4f9b-a306-8531faa51271)

最终还是无法预览：
![微信图片_20241006140757](https://github.com/user-attachments/assets/ccd6c791-8c81-42de-87ec-9700444e5ced)

折腾的好痛苦，求大佬帮忙指明，谢谢。

之前在服务器路径/etc/nginx/conf.d下写了个配置文件onlyoffice.conf：
[https://s21.ax1x.com/2024/10/06/pA84utA.png](url)

用nginx搞不好又换的caddy。人都麻了

**Comment by pongfcnkl** [comment]
only.103006.xyz都访问不上去

**Comment by Simonzhang8** [comment]
看这个：https://github.com/alist-org/alist/discussions/3255#discussion-4822804

**Comment by bigwatermelon8** [comment]
不用折腾反向代理，用onlyoffice自带的的解析就行了。你把view.html文件放到welcome文件夹下面。手动能够访问到就可以了。解析的地址就放这个URL就行。我就是这样搞得，Nginx折腾好就都不行，这才想出来这个办法，核心思想就是要访问到view文件

**Reply by wemio** [reply]
<img width="315" alt="image" src="https://github.com/user-attachments/assets/6a9fdae5-da70-44b5-bb0e-ae2a8e08fcb6" />
提示令牌格式不正确。
